generic/tests/netperf.py

```diff
@@ -95,28 +95,30 @@ def mtu_set(mtu):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
-
-    if len(params.get("nics", "").split()) > 1:
-        session = vm.wait_for_login(nic_index=1, timeout=login_timeout)
-    else:
-        session = vm.wait_for_login(timeout=login_timeout)
-
     config_cmds = params.get("config_cmds")
     if config_cmds:
         for config_cmd in config_cmds.split(","):
             cmd = params.get(config_cmd.strip())
+            session = vm.wait_for_serial_login(timeout=login_timeout)
```

Comment:
@quanwenli Would you please check the above comment and maybe update accordingly? Thanks.

Reply:
I find there is a clear_win_driver_verifier function in virttest/utils_test/qemu/__init__.py, do we need to use it directly ?

Reply:
@quanwenli I suppose we can, the function seems work exactly the same way as @xiagao described. @xiagao Please correct me if I was wrong. Thanks.

Reply:
yes,it looks good to me.

Reply:
@vivianQizhu I find two problems if we use clear_win_driver_verifier. 

1. it need to list the driver list  in  _check_driver_verifier, but in my tests, I do not care which driver verifier need to clear, need reset all the drivers, so It's hard to give dirver list. 
2.  there is once reboot_vm in clear_win_driver_verifier, in netperf.py, there is "reboot_after_config", so it may be need reboot_vm twice in testing. 

Base on it, it seems can not use fucntion  clear_win_driver_verifier. so mybe I need to update base on my original design.

Reply:
@quanwenli I see, both okay for me.

Reply:
@vivianQizhu, the current code is already meet my requirement, for @xiagao suggestion, it's hard to update it in config_cmds judge，the config_cmds is universal for all windows command, we do not need to judge it for verifier output I think. 

And the since it's logging.info(o), the results are shown in debug, if the driver verifier is not settting, we can check the debug log.

Reply:
@quanwenli I get your point, while I suppose we'd better check the output of the "verifier /querysettings" is `Verified Drivers: None` or at lease does not include `netkvm` before reset it. We need to consider the business correction before how we implement it in code. If we can not make it inside `config_cmds` loop(although I guess we probably can), then we might have a piece of code outside it to specifically handle the driver disabling.

Reply:
@vivianQizhu not only with netkvm.sys, but all *.sys from developer's requirement. so we do not need to check "Verified Drivers: None", but need check there is no *.sys in output of  "verifier /querysettings".

Reply:
Yes @quanwenli I think we are in the same line, we need to check there is nothing following "Verified Drivers: " in the output of "verifier /querysettings".

Reply:
@vehrlich , can you help review the latest update. thanks